rest/routes/deepCopyJSON_test.go

```diff
@@ -0,0 +1,53 @@
+package routes
+
+import (
+	"testing"
+
+	"github.com/RedHatInsights/chrome-service-backend/rest/models"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDeepCopyJSON(t *testing.T) {
```

Comment:
@JetyAdam we are missing tests for the `FilterWidgetMappingHeaderLink` function - which I think is the riskier function as it is performing the modifications on the mapped widget object.

Since this PR has been out there for some time I'll confirm everything locally - but I'd like to follow up with another test